pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -451,6 +453,12 @@ var _ = Describe("Converter", func() {
       <driver name="qemu" type="raw" iothread="1"></driver>
       <alias name="ua-pvc_block_test"></alias>
     </disk>
+    <filesystem type="mount">
```

Comment:
Are there any drawbacks to using type="mount" here. I'm curious why we had to go with .iso for exposing the configmap and secret disks.

Reply:
good question, I just looked at the dom xml options, and found this as the most appropriate way to go. Maybe @mfranczy can explain?

Reply:
According to [design doc](https://docs.google.com/document/d/1TcxytalN8CLVnoP7IHh2mFx-xL197w7_obNLUe_z9c0/edit#heading=h.fyb6k0pywmjg) ```iso``` format has been chosen because of 

> The disk will be format with the read-only ISO-9660 file-system (chosen because of it’s wide guest support) and will use the same file-system hierarchy as the Kubernetes Secret/ConfigMap Volumes.

However, I created a trello task to check ```9p``` (few weeks ago), because we need to provide update propagation functionality for configmap and secrets. I don't know about any drawbacks, this is what I wanted to check (maybe it's good enough and should be used as first)

Reply:
9p has a to narrow guest support.

Reply:
> However, I created a trello task to check 9p (few weeks ago), because we need to provide update propagation functionality for configmap and secret

We might end up needing to support iso and 9p. Since guest support is limited, maybe we can make a enabling 9p support for these disks a value that's toggled in the domianSpec feature section on the VMI.

For this PR the right approach is to start with iso for now.  If token rotation becomes a thing within the pod, It's possible in the future we might encounter something similar to the config-map/secret issue with regards to change propagation.

Reply:
I'm close to your position, @davidvossel - we probably want to enable 9p at some point. After all it has some benefits on supported platforms (i.e. "live updates"), but by then we might also have nfs-over-virtio which will also be a first class citizen on linux at least.

Now, anyway - to see how valuable features are to users, we can start with the iso9660 approach, and if it flies we can improve it.
If we add it in one place, then we probably want to use it everywhere where we used iso9660 (secret/cm, cloudinit, here)